iavl/iavl_proof2.go

```diff
@@ -58,27 +60,28 @@ func (p *PathToKey) isRightmost() bool {
 	return true
 }
 
-type innerPathPair struct {
-	left  []IAVLProofInnerNode
-	right []IAVLProofInnerNode
+func (p *PathToKey) pop() *PathToKey {
```

Comment:
Haha. I've renamed it to `dropRoot`, since it is the root that it is dropping, and renamed the other function to make it clear. See c3e77933c170553be7e1934b8145baf7d88f5c5d